front/app/containers/IdeasNewPage/messages.ts

```diff
@@ -358,4 +358,20 @@ export default defineMessages({
     defaultMessage:
       'The issue description must be more than {limit} characters long',
   },
+  leaveSurveyConfirmationQuestion: {
+    id: 'app.containers.IdeasNewPage.leaveSurveyConfirmationQuestion',
+    defaultMessage: 'Are you sure you want to leave this survey?',
+  },
+  leaveSurveyMessage: {
+    id: 'app.containers.IdeasNewPage.leaveSurveyMessage',
+    defaultMessage: 'Your answers won`t be saved.',
```

Comment:
A back quote in a sentence is not the right quote. It should be a single quote. Example: `"app.components.ConfirmationModal.didntGetAnEmail": "Didn't receive an email?",`